userdocs/src/usage/vpc-cluster-access.md

```diff
@@ -68,6 +68,10 @@ To update the restrictions using a `ClusterConfig` file, set the new CIDRs in `v
 eksctl utils set-public-access-cidrs -f config.yaml
 ```
 
+!!!warning
+    If setting `publicAccessCIDRs` and creating nodegroups `privateAccess` should be set to `true` to avoid
+    `context deadline exceeded` errors due to the nodes being unable to join the cluster.
```

Comment:
I was more thinking in the way of WHY it's failing or what it will cause. So something like

```
When setting `publicAccessCIDRs` and creating nodegroups, `privateAccess` must be set to `true` otherwise nodes will be unable to join the cluster due to their ip addresses missing from public access CIRDs.
```

Because when private access is enabled, the nodes will join in a different way. But that's stretching it. :) So the context deadline is irrelevant, an implementation detail. The important part is what happens and why if that makes sense :)

Reply:
The reason I was leaving the context deadline in there is that that's the error people will see (as far as I can tell it's the only consistent error, I've sometimes seen "node unable to join cluster" but not always; I assume depending on where the deadline fires).

I can add in more detail as to what's going on but I think including the error is important as I couldn't find anything about this when I hit it

Reply:
I understand. In that case, add it to the troubleshooting section? The error is a detail which I think is not something you would care about while reading the documentation and going through it step by step, right? Especially if it's not always context deadling exceeded but node unable to join. :)

Reply:
`deadline exceeded` is always the error I've seen. I've just seen different errors previously to it (e.g. [here](https://gist.github.com/SamLR/a88149789fe46cc658fa8f27b0949f74) it errors with `unexpected status "CREATE_IN_PROGRESS"`; other times I've seen `unable to join`).

I can move it if you think that's a more useful location for it but I'd assume most people will encounter the `publicAccessCIDRs` option here so it makes sense to have the warning here where they'll find it before they trigger the error than providing it to them after.

Reply:
Meh, it's fine. :D